### ISSUE TITLE
Not able to follow some groups on group public page

### ISSUE THREAD

**Issue by pc-beast** [fossasia/open-event-frontend#8135]
We are not able to follow some groups with follow button on the group public page.
Compare: https://eventyay.com/g/101

![image](https://user-images.githubusercontent.com/56963647/143383534-22d02a89-777c-4120-a591-d54a6cef61e5.png)

![image](https://user-images.githubusercontent.com/56963647/143383488-5f6ff478-6616-476b-8861-287c228a7284.png)

**Comment by subhamgcon** [comment]
Hey I tried to compare this bug but its working me.

**Comment by mariobehling** [comment]
@subhamgcon Yes, this is the confusing thing. Sometimes works and sometimes not for some accounts. We discuss these kind of details also in our regular meetings on Tuesdays and Thursdays. Would be great if you join. You get the latest info on the meetings in our Gitter channel https://gitter.im/fossasia/open-event-frontend Thank you.

**Comment by subhamgcon** [comment]
> @subhamgcon Yes, this is the confusing thing. Sometimes works and sometimes not for some accounts. We discuss these kind of details also in our regular meetings on Tuesdays and Thursdays. Would be great if you join. You get the latest info on the meetings in our Gitter channel https://gitter.im/fossasia/open-event-frontend Thank you.

Thank you, I will definitely join from the next meeting!
